build.gradle

```diff
@@ -123,4 +124,8 @@ task renderEN(type: RenderCurriculumTask,
 
 apply from: 'scripts/generateTermTables.gradle'
 
+// Executed by: ./gradlew -q sortTermsAlphabetically
```

Comment:
Since this is now triggered by the renderDE task anyway, do we need this separate explanation comment?

Reply:
Just a reminder for gradle newbies like me how to run this task standalone. I will move it to the task internal documentation.